mobile_resources_commit

```diff
@@ -1 +1 @@
-e9c1e9da272e37e63bc8e53b55d3206f42e1ae32
```

Comment:
Is this update intentional here?

Reply:
good question - yes, I updated to use `ENAPIVersion`: `2`